chains/137.json

```diff
@@ -0,0 +1,564 @@
+[
+  {
+    "chainId": 137,
+    "name": "Ether - PoS",
+    "symbol": "ETH",
+    "decimals": 18,
+    "address": "0x7ceb23fd6bc0add59e62ac25578270cff1b9f619",
+    "logoURI": "../images/eth.svg"
+  },
+  {
+    "chainId": 137,
+    "name": "USD Coin",
+    "symbol": "USDC",
+    "decimals": 6,
+    "address": "0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
+    "logoURI": "../images/usdc.svg"
+  },
+  {
+    "chainId": 137,
+    "name": "Tether USD",
+    "symbol": "USDT",
+    "decimals": 6,
+    "address": "0xc2132d05d31c914a87c6611c10748aeb04b58e8f",
+    "logoURI": "../images/usdt.svg"
+  },
+  {
+    "chainId": 137,
+    "name": "Dai - PoS",
+    "symbol": "DAI",
+    "decimals": 18,
+    "address": "0x8f3cf7ad23cd3cadbd9735aff958023239c6a063",
+    "logoURI": "../images/dai.svg"
+  },
+  {
+    "chainId": 137,
+    "name": "Aave",
+    "symbol": "AAVE",
+    "decimals": 18,
+    "address": "0xd6df932a45c0f255f85145f286ea0b292b21c90b",
+    "logoURI": "../images/aave.svg"
+  },
+  {
+    "chainId": 137,
+    "name": "ChainLink Token",
+    "symbol": "LINK",
+    "decimals": 18,
+    "address": "0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39",
+    "logoURI": "../images/link.svg"
+  },
+  {
+    "chainId": 137,
+    "name": "Wrapped BTC",
+    "symbol": "WBTC",
+    "decimals": 8,
+    "address": "0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6",
```

Comment:
PoS

Reply:
Thinking about it, we could use a list extension here to label these as PoS if we want